### PR TITLE
Disambiguate icon for daily builds on Windows

### PR DESCRIPTION
**Pull request by bergmeister**
Change icon of `pwsh.exe` for daily builds on Windows to [this](https://raw.githubusercontent.com/PowerShell/PowerShell/master/assets/Powershell_av_colors.ico) icon.
Some code had to be borrowed from build.psm1 because this script has to be self contained in case it gets executed by only downloading this file via the published download link [here](https://twitter.com/Steve_MSFT/status/930585082451992576).
Tested on Windows 10.0.16299 (Fall Creators Update) with WMF 5.1

### CODE REVIEW

File: tools/install-powershell.ps1 [hunk 1]
```diff
@@ -206,6 +206,19 @@ try {
...
+    # Edit icon to disambiguate daily builds.
+    if ($IsWinEnv -and $Daily.IsPresent) {
+        if (-not (Test-Path "~/.rcedit/rcedit-x64.exe")) {
+            Write-Verbose "Install RCEdit for modifying exe resources"
+            $rceditUrl = "https://github.com/electron/rcedit/releases/download/v1.0.0/rcedit-x64.exe"
+            New-Item -Path "~/.rcedit" -Type Directory -Force > $null
+            Invoke-WebRequest -OutFile "~/.rcedit/rcedit-x64.exe" -Uri $rceditUrl
+        }
+
+        Write-Verbose "Change icon to disambiguate it from a released installation"
```
Comment (daxian-dbw) [inline tools/install-powershell.ps1]: Same here. Please add `-Verbose`.

**Review by daxian-dbw — COMMENTED** [review]
Thanks for the improvement! I left a few comments.

File: tools/install-powershell.ps1 [hunk 2]
```diff
@@ -206,6 +206,19 @@ try {
         }
     }
 
+    # Edit icon to disambiguate daily builds.
+    if ($IsWinEnv -and $Daily.IsPresent) {
```
[the remark this answers is not in the document]
Comment (bergmeister) [reply tools/install-powershell.ps1]: Ok.

File: tools/install-powershell.ps1 [hunk 3]
```diff
@@ -206,6 +206,19 @@ try {
...
+        if (-not (Test-Path "~/.rcedit/rcedit-x64.exe")) {
+            Write-Verbose "Install RCEdit for modifying exe resources"
```
[the remark this answers is not in the document]
Comment (bergmeister) [reply tools/install-powershell.ps1]: Ok. Done.

File: tools/install-powershell.ps1 (hunk 1 above)
Comment (bergmeister) [reply tools/install-powershell.ps1]: Done as well now.

**Review by daxian-dbw — APPROVED** [review]

**Comment by daxian-dbw** [comment]
@SteveL-MSFT Can you please take a look?

**Review by SteveL-MSFT — APPROVED** [review]
LGTM

**Pull request merged**